layouts/shortcodes/release-binaries.html

```diff
@@ -1,5 +1,13 @@
 <!-- Fetch release_binaries.json from kubernetes-sigs/downloadkubernetes to render in table -->
 {{ $response := getJSON "https://raw.githubusercontent.com/kubernetes-sigs/downloadkubernetes/master/dist/release_binaries.json" }}
```

Comment:
Really, we should switch this to use `transform.GetRemote`; see https://gohugo.io/functions/resources/getremote/#remote-data for an example that includes error handling. Then I don't think we need to edit the `hugo.toml` to ignore `error-remote-getjson`.

(If we switch, we can still change the error handling based on which environment we're in)